articlequality/extractors/ptwiki.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import re
+
+from .extractor import TemplateExtractor
+
+logger = logging.getLogger(__name__)
+
+
+def from_template(template):
+    project_name = normalize_project_name(template.name)
+    if template.name == "marca de projeto":
+        labels = extract_labels(project_name)
+        if len(labels) > 1:
+            for project, label in labels[1:]:
+                yield (project, label)
+
+
+PROJECT_LABEL = re.compile(r"([^\|\{\{\}\}]+)\|([0-5\*])", re.I)
+
+
+def extract_labels(template):
+    return [(label.group(1),
+             label.group(2)) for label in re.finditer(PROJECT_LABEL, (template))]
```

Comment:
template need to be converted to a string.  E.g. `str(template)`

Reply:
I did that before, which raised the key error